### PR TITLE
feat avatar & badge

### PR DESCRIPTION
**Pull request by sudipkundu999**
File structure:
components>avatar>avatar.css => contains CSS for avatar only
docs>component-presentation>avatar-presentation.css => contains CSS for presenting the avatar on the documentation site

### CODE REVIEW

File: Components/avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
+.avatar{
+    --size:50px;
```
Comment (iRohitGaur) [inline Components/avatar/avatar.css:2]: try to work with rem instead of fixed px

File: Components/avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: relative;
+}
+
+.avatar img{
+    height: inherit;
+    width: inherit;
+    border-radius: 100%;
```
Comment (iRohitGaur) [inline Components/avatar/avatar.css]: why 100%? 50% is enough if you are aiming for circle

File: Components/avatar/avatar.css [hunk 3]
```diff
@@ -0,0 +1,36 @@
+.avatar{
+    --size:50px;
+    height: var(--size);
+    width: var(--size);
+    border-radius: 100%;
+
+    color: white;
+    background:grey;
```
Comment (zoef-yt) [inline Components/avatar/avatar.css:8]: suggestion use variables here

File: Components/badge/badge.css [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    min-width:1.2rem;
+    width: max-content;
+    min-height:1.2rem;
+    height: max-content;
+
+    border-radius:100%;
+    border: 1px solid white;
+    background: red;
+    position: absolute;
+    right:0
```
Comment (iRohitGaur) [inline Components/badge/badge.css]: missing a semicolon here. similar occurrences in multiple places

**Review by zoef-yt — COMMENTED** [review]
Good class name. Nice consistent code. Good to merge.

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge

**Review by amartya-rs — APPROVED** [review]
Try to use color variables. Otherwise, code looks good.

File: Components/avatar/avatar.css (hunk 3 above)
Comment (sudipkundu999) [reply Components/avatar/avatar.css:8]: I will use variables eventually. Right now I'm not sure about the colors so I thought to go with basic colors and when the component library becomes complete replace all with variables.

File: Components/avatar/avatar.css (hunk 2 above)
Comment (sudipkundu999) [reply Components/avatar/avatar.css]: Thanks, Rohit! I always used to do 100%

File: Components/avatar/avatar.css (hunk 1 above)
Comment (sudipkundu999) [reply Components/avatar/avatar.css:2]: Done

**Pull request merged**